.github/workflows/build.yml

```diff
@@ -73,6 +73,14 @@ jobs:
           name: ${{ matrix.version }}_previous
           path: build/${{ matrix.version }}/report.json
 
+      # Build and link non-matching files to verify lack of errors
+      - name: Build non-matching
+        run: |
+          python configure.py --non-matching --map \
```

Comment:
I don't thin we need `--map` here. It slows down the build a lot. I think we only use it above for some sort of progress/metrics calculation.